app/src/main/java/com/example/cooky/data/local/dao/MealPlanDAO.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.cooky.data.local.dao
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+import com.example.cooky.data.remote.response.MealPlanResponse
+
+@Dao
+interface MealPlanDAO {
```

Comment:
```suggestion
interface MealPlanDao {
```
Tên viết tắt nếu có 2 chữ thì có thể để (Vd: IO, UI...), 3 chữ trở lên thì coi như 1 từ.

Reply:
oke cái này tôi làm 1 pull sửa sau vì mấy cái DAO kia tôi cũng viết thế